pkg/apiserver/server.go

```diff
@@ -37,6 +38,7 @@ var (
 		fx.Provide(
 			newEngine,
 			newAPIRouter,
+			core.NewKubeWorkflowRepository,
```

Comment:
We haven't provided the `client.Client` instance in `fx.provide`. I think this registration will fail.

Reply:
BTW, all APIs use `github.com/chaos-mesh/chaos-mesh/pkg/clientpool` as the kube client. Should workflow API also use it?

Reply:
yeah, we should use the `clientpool`, I will fix it ASAP.😅

Reply:
Seems we don't need to provide this currently.